app/views/courses/show.html.erb

```diff
@@ -127,7 +127,7 @@
     <%= render :partial => 'assignments/syllabus_right_side', :locals => {:course_home => true} %>
 hi there
   <% else %>
-    <% locals = {:title => t("Coming Up"), :contexts_to_link_to => @context, :upcoming => true, :period => :one_week} %>
+    <% locals = {:title => t("Upcoming Events"), :contexts_to_link_to => @context, :upcoming => true, :period => :one_year} %>
```

Comment:
I don't see how this filters down to just events?  Wouldn't this pick up assignments too?